tox.ini

```diff
@@ -15,6 +16,7 @@ deps =
     dj4: Django>=4,<5
     wagtail215: wagtail>=2.15,<2.16
     wagtail216: wagtail>=2.16,<2.17
+    wagtail3: wagtail>=3.0,<4.0
```

Comment:
**nitpik** I wonder if using `wagtail30: wagtail>=3.0,<4.0` might be better. Although I can see why you did this looking at the current Django versions

Reply:
probably keep it consistent with `dj4` is better